dataprofiler/labelers/character_level_cnn_model.py

```diff
@@ -653,7 +670,7 @@ def fit(
             history[metric_label] = model_results[i]
 
         if val_data:
-            f1, f1_report = self._validate_training(val_data)
+            f1, f1_report = self._validate_training(val_data)  # type: ignore
```

Comment:
let's see how we might be able to remove the `#type: ignore` and maybe remove `if val_data` on line 672

Reply:
removed redundant `if val_data`